modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -219,18 +255,148 @@ sudo apt install phpmyadmin
+[source,console]
+----
+resource:foreign_path local_path nfs (<your options>),_netdev
+----
+--
+
+* Make sure that all mounts in `/etc/fstab` are mounted by running:
++
+--
+[source,console]
+----
+sudo mount -a
+----
+--
+
+* Run the following command to list mounts which must be up first:
++
+--
+[source,console]
+----
+systemctl list-units | grep -nP "\.mount"
+----
+
+You should see lines printed to the console.
+Look for the mount you want to be up in the command's output. 
+
+[source,console]
+----
+<folder.mount>
+  loaded active mounted <local_path>
+----
+where `<folder.mount>` and `<local_path>` are examples!
+--
+
+* Edit the service you want to change:
++
+--
+[source,console]
+----
+sudo systemctl edit <name>.service
+----
+
+Add the following directive in the editor opened using your chosen `folder.mount` from above:
+
+[source,console]
+----
+[Unit]
+After=folder.mount
+----
+
+You can add more than one dependency if needed by separating them by blank spaces.
+This procedure keeps `<name>.service` in its original state but makes it possible to override the current setup with new parameters.
+It automatically creates a directory in `/etc/systemd/system`, named `<name>.service.d`, and a file in that directory called `override.conf`.
+In the example above, the parameter is added to the existing list of parameters of the `After` directive.
+
+For more details please read section {overriding-vendor-settings-url}[Example 2. Overriding vendor settings]
+
+Please keep the following points in mind, regarding if `<name>.service` is linked or not:
+
+** If the file is linked from `/lib/systemd/system`, it is for packaged unit files.
+They will get overwritten when Systemd (or whatever package provides them) is upgraded.
+
+** If the file originates in `/etc/systemd/system`, it is for your own and customised unit files.
+Unit files you place in here will override the package-provided file and will not be replaced on upgrade.
+
+It is recommended to keep things simple and future proof by creating an override file via `systemctl edit`.
+--
+
+* Run the following command to apply your changes:
++
+--
+[source,console]
+----
+sudo systemctl daemon-reload
+----
+--
+
+* Check if `<name>.service` has been properly added:
++
+--
+[source,console]
+----
+sudo systemctl show <name>.service | grep "After="
+----
+`folder.mount` should be part of the parameter list.
+--
+
+* Restart your service by invoking:
++
+--
+[source,console]
+----
+sudo system <your service name> restart
+----
+--
+
+=== Disabling Transparent Huge Pages (THP)
+
+{transport-huge-pages-url}[Transparent Huge Pages] should be disabled when using databases.
+This is not only true when using Redis but also for MariaDB. 
+For more information read: {disabling-thp-url}[Why THP (Transparent Huge Pages) are not recommended for Databases].
+
+To disable Transparent Huge Pages, follow these steps:
+
+* Create in `/etc/systemd/system` a file like `disable-thp.service` and add this content:
```

Comment:
Instead of "add this content", I suggest "add the following content".

Reply:
fixed